lib/perl/Genome/Config/Profile/Item.pm

```diff
@@ -131,26 +132,29 @@ sub has_model_for {
 sub _create_allocation_for_file {
     my $self = shift;
     my $file_to_store = shift;
+    my $skip_allocation_path_creation = shift // 1;
```

Comment:
Maybe this becomes:
```perl
shift // !$ENV{UR_DBI_NO_COMMIT}
```
or even:
```perl
(shift // 1) && !$ENV{UR_DBI_NO_COMMIT}
```
if we want no-commit to disable the option regardless of whatever was passed in.